src/JuxtaposeApplication.jsx

```diff
@@ -48,7 +48,7 @@ export default class JuxtaposeApplication extends React.Component {
                    const source = extractSource(assetCtx.sources);
                    const annotation = extractAnnotation(assetCtx, assetData.annotationId);
                    
-                   if (e.detail.caller.type === 'spine') {
+                   if (e.detail.caller === 'spine') {
```

Comment:
Right, the error is here: https://github.com/ccnmtl/juxtapose/blob/master/src/SpineVideo.jsx, I'll update.